source/math/FloatVectorList.ooc

```diff
@@ -21,15 +21,20 @@ import FloatComplexVectorList
 
 FloatVectorList: class extends VectorList<Float> {
 	init: func ~default {
-		this super()
+		super()
 	}
 	init: func ~heap (capacity: Int) {
 		super(capacity)
 	}
 	init: func ~fromVectorList (other: VectorList<Float>) {
-		this super(other _vector)
+		super(other _vector)
 		this _count = other count
 	}
+	init: func ~withValue (capacity: Int, value: Float) {
+		super(capacity)
+		for (i in 0 .. capacity)
+			this add(value)
```

Comment:
Should I add an extra check for this? Then I have to set `_count` manually instead.